### DISCUSSION TITLE
Request--> Wraith: the Oblivion Character Sheet

### DISCUSSION THREAD

**Discussion by hammiah** [JohanFalt/Foundry_WoD20#519] [Q&A] [answered]
Hello!

Congratulations to the great job on this module!

Is there some chance of a Wraith character sheet for this module?

**Comment by JohanFalt** [comment, marked as answer]
Yes Wraith is planned for the next big release v2.5 still a couple of months away.

You can follow the development here:

https://github.com/JohanFalt/Foundry_WoD20/milestone/15
